manifests/prometheus/servingCertsCABundle.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+data:
+  service-ca.crt: ""
+kind: ConfigMap
```

Comment:
is this exactly the same CM that is in the manifests/prometheus/list.yaml?

Reply:
Yes. All Prometheus manifests are written to `manifests/prometheus/` and one templateable list manifest is written to `manifests/prometheus/list.yaml`. This list includes all of the manifests. Same goes for the Telemeter server.

Reply:
I see, thanks for explanation